ratelimits/transaction.go

```diff
@@ -38,23 +41,32 @@ func newRegIdBucketKey(name Name, regId int64) string {
 	return joinWithColon(name.EnumString(), strconv.FormatInt(regId, 10))
 }
 
-// newDomainBucketKey validates and returns a bucketKey for limits that use the
-// 'enum:domain' bucket key format.
-func newDomainBucketKey(name Name, orderName string) string {
-	return joinWithColon(name.EnumString(), orderName)
+// newStringBucketKey validates and returns a bucketKey for limits that use
+// the 'enum:identValue' or 'enum:domainOrCIDR' bucket key formats.
+func newStringBucketKey(name Name, value string) string {
+	return joinWithColon(name.EnumString(), value)
+}
+
+// NewRegIdIdentValueBucketKey validates and returns a bucketKey for limits that
+// use the 'enum:regId:identValue' bucket key format. This function is exported
+// for use in ra.resetAccountPausingLimit.
+func NewRegIdIdentValueBucketKey(name Name, regId int64, orderIdent identifier.ACMEIdentifier) string {
+	return newRegIdStringBucketKey(name, regId, orderIdent.Value)
 }
 
-// NewRegIdDomainBucketKey validates and returns a bucketKey for limits that use
-// the 'enum:regId:domain' bucket key format. This function is exported for use
-// in ra.resetAccountPausingLimit.
-func NewRegIdDomainBucketKey(name Name, regId int64, orderName string) string {
-	return joinWithColon(name.EnumString(), strconv.FormatInt(regId, 10), orderName)
+// newRegIdStringBucketKey validates and returns a bucketKey for limits that use
```

Comment:
As long as we're updating this comment, let's fix it to remove "validates", which it doesn't do anymore. Same above.